chapters/equations.tex

```diff
@@ -600,12 +590,14 @@ \section{Events and Synchronization}\doublelabel{events-and-synchronization}
 the value of the expression can only be changed at event instants. If
 the evaluated expression is inconsistent with the buffer, that will
 trigger an event and the buffer will be updated with a new value at the
-event instant {[}\emph{in other words, a root finding mechanism is
-needed which determines a small time interval in which the expression
-changes its value; the event occurs at the right side of this
-interval}{]}. During continuous integration event generation expression
+event instant. During continuous integration event generation expression
 has the constant value of the expression from the last event instant.
 
+\begin{nonnormative}
+in other words, a root finding mechanism is needed which determines a small time interval in which the expression changes its value; the event occurs
```

Comment:
This should probably start with a capital letter to be more freestanding.

Reply:
I took out the entire _in other words_.

I'm a bit concerned about non-normative nature of this statement, however.  The _the event occurs…_ part doesn't look like typical non-normative content.  Can it be change to something less definite, or should the entire thing become normative?  (I'm hoping it doesn't become normative, as I'm not convinced it makes sense to define the event to occur at the right side.)